src/node/plugin.ts

```diff
@@ -49,4 +49,8 @@ export const loadPlugins = async (httpServer: HttpServer, args: Args): Promise<v
       logger.warn(error.message)
     }
   }
+
+  if (process.env.PLUGIN_DIR) {
+    await loadPlugin(process.env.PLUGIN_DIR, httpServer, args)
```

Comment:
Perhaps it'd be better to call `_loadPlugins` with this directory instead to allow multiple external plugins in an arbitrary directory?

Reply:
I had a very similar thought but I wasn't sure how to make that work with the current x11wasm structure. Currently it's just a `code-server` directory in the `x11wasm` repo so it would try to load every directory in there. Maybe we could have a `code-server-plugins` directory inside `x11wasm`?

Reply:
Another thought is to require symlinking into `plugins` in the `code-server` directory instead of an environment variable. Or maybe make `PLUGIN_DIR` able to contain multiple paths.